components/KiboOrderItem.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="orders">
+    <div class="order-item-container">
+      <div class="order-item">
+        <div class="order-item__left">
+          <div class="order-item__info">{{ order.submittedDate }}</div>
+          <div class="order-item__description">{{ order.products }}</div>
+          <div class="order-item__info">{{ order.total }}</div>
+          <div class="order-item__status">{{ order.status }}</div>
+        </div>
+        <div class="order-item__right">
+          <SfChevron class="sf-chevron--right" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import { SfChevron } from "@storefront-ui/vue"
+import { defineComponent } from "@vue/composition-api"
+
+export default defineComponent({
+  name: "KiboOrderItem",
+  components: {
+    SfChevron,
+  },
+  props: {
+    order: {
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  setup() {},
+})
+</script>
+
+<style lang="scss" scoped>
+.order-item-container {
+  border: none;
+  border-bottom: 1px solid #eaeaea;
+}
+
+.order-item {
+  display: flex;
+  flex-direction: row;
+  margin: 10px 0 10px 23px;
+
+  &__left {
+    display: flex;
+    flex-direction: column;
+    flex: 6;
+    line-height: 19px;
+    gap: 5px;
+  }
+
+  &__right {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    justify-items: flex-end;
+    flex: 1;
+  }
+
+  &__info {
+    color: #2b2b2b;
+    font-family: var(--font-family--primary);
```

Comment:
As we are defining primary font globally, it should apply it automatically. Do we need to mention it explicitly?